dev/reset-test-only.sh

```diff
@@ -10,6 +10,7 @@ if [ $# -ne 2 ]; then
     psql -Upostgres -f openchpl_preload.sql openchpl_test
     psql -Upostgres -f openchpl_api-key.sql openchpl_test
     psql -Upostgres -f openchpl_quartz.sql openchpl_test
+	psql -Upostgres -f openchpl_ff4j.sql openchpl_test
```

Comment:
It's tiny, but these various "different indentation" things irk the heck out of me